tools/appliance/systemvmtemplate/scripts/apt_upgrade.sh

```diff
@@ -53,11 +53,17 @@ function apt_upgrade() {
 
   rm -fv /root/*.iso
   apt-get -q -y update
+
+  # Remove unused non-default kernel
   apt-get -q -y upgrade
   apt-get -q -y dist-upgrade
+  apt-get -q -y upgrade -t buster-backports
+  apt-get -q -y dist-upgrade -t buster-backports
+
   apt-get -y autoremove --purge
   apt-get autoclean
   apt-get clean
+  reboot
 }
```

Comment:
@rhtyd might 'reboot' be an issue in packer processing ?

Reply:
No see template.json